pkg/apis/core/types_shoot.go

```diff
@@ -423,6 +423,8 @@ type KubeAPIServerConfig struct {
 	// of the API server should be allowed (flag `--anonymous-auth`).
 	// See: https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/
 	EnableAnonymousAuthentication *bool
+	// EventTTL controls the amount of time to retain events.
```

Comment:
We can mention the default value in the doc string (it can be useful for someone reading the generated API reference).

```suggestion
	// EventTTL controls the amount of time to retain events.
	// Defaults to 1h.
```